WebDevStudios/nextjs-wordpress-starter
# Feature/refactor displayblock

Closes N/A

### Description

* Refactor `displayBlock.js`, move imports into case statement
* Fully remove LazyBlocks from the frontend
* Remove all references to ACF from the documentation
* Add `NEXTAUTH_SECRET` to .env.sample
* Rename block handler directory to `core`
* Remove CPTUI from backend `composer.json`

### Screenshot

N/A

### Verification

How will a stakeholder test this?

1. Pull this branch
2. Verify that block still display on the frontend
3. Verify that anything related to LazyBlocks has been fully removed from the frontend
4. Verify that anything related to ACF has been removed from the documentation
